### PR TITLE
2124

### PR DESCRIPTION
**Pull request by midhun1998**
Solved issue #2124 and now the zip file is being saved as `ums_dbg_YYYY-MM-DD-hh:mm.zip`. (Example `ums_dbg_2020-06-24-21:03.zip`).

### CODE REVIEW

**Comment by ExSport** [comment]
Hi @midhun1998 
Colon is illegal character on Windows as it is part of drive letter.
Please replace it with - or _ 😎
Thank you

**Comment by midhun1998** [comment]
Got it. Will work on that. :smiley:

**Comment by midhun1998** [comment]
Hey @ExSport 
I have changed the **:** to **-**. :+1:

**Comment by ExSport** [comment]
@midhun1998  TY

File: src/main/java/net/pms/newgui/DbgPacker.java [hunk 1]
```diff
@@ -53,7 +56,10 @@ public DbgPacker() {
 			zippedLogFile = PMS.getConfiguration().getDefaultLogFilePath();
 		}
 		if (!zippedLogFile.isEmpty()) {
-			zippedLogFile = FileUtil.appendPathSeparator(zippedLogFile) + "ums_dbg.zip";
+			DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd-HH:mm");
```
Comment (SubJunk) [inline src/main/java/net/pms/newgui/DbgPacker.java:59]: Not sure if the fix for this got reverted on the way, but as @ExSport said, the colon breaks Windows support

Comment (midhun1998) [reply src/main/java/net/pms/newgui/DbgPacker.java]: @SubJunk Fixed it Thanks. :smile:

**Comment by SubJunk** [comment]
Great work, thank you!

**Pull request merged**